### DISCUSSION TITLE
Mixed content hosting connected with cloudfire

### DISCUSSION THREAD

**Discussion by offerstyle** [wintercms/winter#713] [Q&A] [answered]
HI
I installed winter cms on the hosting and when I log in to the admin panel, a (Mixed content) problem is displayed in the console
ScreenCapture
https://disk.yandex.com.am/i/pFkyTgwKn3U-og
hosting connected with cloudfire

Thanks in advance

**Comment by LukeTowers** [comment, marked as answer]
Screenshot didn't load for me, check your developer tools to see what resources are not being loaded over HTTPS and fix them. Probably just an error with your configuration, try setting your app.url config value to include the HTTPS link.

**Reply by offerstyle** [reply]
http://interbank-aruba.org/account/modules/backend/widgets/mediamanager/assets/js/mediamanager-browser-min.js?v1'

**Reply by offerstyle** [reply]
https://snipboard.io/AhxYez.jpg

**Reply by bennothommo** [reply]
Make sure you're using [trusted proxies](https://wintercms.com/docs/setup/configuration#trusted-proxies) as well if you're running through a CloudFlare proxy service.

**Reply by LukeTowers** [reply]
@offerstyle try using the new [Winter.Cloudflare plugin](https://github.com/wintercms/wn-cloudflare-plugin)
